cloud/terraform/aws/dependencies/data.tf

```diff
@@ -0,0 +1,10 @@
+data "aws_caller_identity" "current" {
+}
+data "aws_eks_cluster" "cluster" {
+	name=var.cluster_name
+}
+data "aws_eks_cluster_auth" "cluster" {
+	name=var.cluster_name
+}
+data "aws_region" "current" {
```

Comment:
It's bizarre that we use both a `data` block and a variable `region`. Seems inconsistent. If we want it as a variable, I'd recommend removing it as a data block.

Reply:
Seems we're using the `data` value to interpolate an ARN, so I think we're just trying to use a more deterministic value rather than allowing user input (the variable) to potentially break the interpolation.

I understand what you mean, though. Let's do a quick call today so we can hash this out.